app/src/common/utils/validation/commonValidators.js

```diff
@@ -43,6 +43,8 @@ export const btsIntegrationName = bindMessageToValidator(
 );
 export const btsProjectKey = bindMessageToValidator(validate.btsProject, 'btsProjectKeyHint');
 export const btsProjectId = bindMessageToValidator(validate.btsProject, 'btsProjectIdHint');
+export const btsUserName = bindMessageToValidator(validate.btsUserName, 'btsUserNameHint');
+export const btsPassword = bindMessageToValidator(validate.btsPassword, 'btsPasswordHint');
```

Comment:
I'm not sure if these integration field validators can be considered as common (Jira is the only plugin that uses such a combination).
Move them to the appropriate component (app/src/comComponents/integrations/integrationProviders/jiraIntegration/jiraConnectionFormFields), as well as the messages for them.